### PR TITLE
Fix dispute card values

### PR DESCRIPTION
**Pull request by bthaile**
Make sure all outcomes are present for categorical and binary markets when market is in disputing.

image

there is still issues with categorical markets when pre-filled stake is added and with bond size on past tentative winning outcome. Tickets have been created.

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 📈

 This link will always point to this PR's latest changes: https://peek.run/kq3hz7j

File: packages/augur-ui/src/modules/common/selection.tsx [hunk 1]
```diff
@@ -160,7 +160,7 @@ class Dropdown extends React.Component<DropdownProps, DropdownState> {
           [Styles.openTop]: openTop,
           [`${activeClassName}`]: showList,
           [Styles.showColor]: showColor,
-          [`${Styles[`showColor-${selected.value + 1}`]}`]: selected && showColor
+          [`${Styles[`showColor-${selected ? selected.value + 1 : 1}`]}`]: selected && showColor
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/common/selection.tsx:163]: should we get rid of the selected check after the colon

File: packages/augur-ui/src/modules/common/form.tsx [hunk 2]
```diff
@@ -2170,7 +2213,7 @@ export class InputDropdown extends Component<
           ))}
         </select>
         <span>
-          <ChevronFlip pointDown={!showList} stroke='white' />
+          <ChevronFlip pointDown={!showList} stroke="white" />
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/common/form.tsx]: should be single quotes

File: packages/augur-ui/src/modules/common/selection.tsx (hunk 1 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/common/selection.tsx:163]: selected can be null. or an object or string or number. a bit crazy! let's leave it for now.

**Review by phoebemirman — APPROVED** [review]

**Pull request merged**